### PR TITLE
Add vision-annotate-images sample function

### PR DESCRIPTION
**Pull request by nauthadr**

### CODE REVIEW

File: vision-annotate-images/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Vision Annotate Images
+
+This function acts as a pass through to the [Cloud Vision JSON API](https://cloud.google.com/vision/docs/request). 
+
+To call this from an app, you must first sign-in using [Firebase Auth](https://firebase.google.com/docs/auth).
+Only users who pass a valid Firebase ID token as a Bearer token in the `Authorization` header of the HTTP request are authorized to use the function.
+
```
Comment (nauthadr) [inline vision-annotate-images/README.md:7]: I'm thinking once the firebase docs are ready we can just link to those as an example of how to use this in an app. What do you think?

Comment (samtstern) [reply vision-annotate-images/README.md:7]: Yep sg

File: vision-annotate-images/functions/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "start": "npm run shell",
+    "deploy": "firebase deploy --only functions",
+    "logs": "firebase functions:log"
+  },
+  "engines": {
+    "node": "10"
+  },
+  "main": "lib/index.js",
+  "dependencies": {
+    "firebase-admin": "^8.10.0",
```
Comment (samtstern) [inline vision-annotate-images/functions/package.json:17]: `firebase-admin` 9+ is the version to use when using Node 10+ (which you are, according to the `engines` field)

File: vision-annotate-images/functions/package.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "firebase-functions": "^3.6.1",
```
Comment (samtstern) [inline vision-annotate-images/functions/package.json:18]: This looks a little old as well.

File: vision-annotate-images/functions/src/index.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import * as functions from "firebase-functions";
+import * as admin from "firebase-admin";
+import vision from "@google-cloud/vision";
+
+admin.initializeApp();
```
Comment (samtstern) [inline vision-annotate-images/functions/src/index.ts:5]: nit: doesn't look like you actually use `firebase-admin` at all in this function?

File: vision-annotate-images/functions/package.json (hunk 2 above)
Comment (nauthadr) [reply vision-annotate-images/functions/package.json]: Done

File: vision-annotate-images/functions/package.json (hunk 3 above)
Comment (nauthadr) [reply vision-annotate-images/functions/package.json]: Updated to 3.11.0

**Review by samtstern — APPROVED** [review]

File: vision-annotate-images/functions/src/index.ts (hunk 4 above)
Comment (nauthadr) [reply vision-annotate-images/functions/src/index.ts]: You're right. For some reason I thought it was required to do initializeApp. I've removed it.

**Pull request merged**